tools/syncer.md

```diff
@@ -490,35 +504,40 @@ Syncer provides the metric interface, and requires Prometheus to actively obtain
 
 #### title: binlog events
 
-- metrics: `irate(syncer_binlog_events_total[1m])`
-- info: the master binlog statistics that has been synchronized by Syncer, including the five major types of `query`, `rotate`, `update_rows`, `write_rows` and `delete_rows`
+- metrics: `rate(syncer_binlog_event_count[1m])`
```

Comment:
seems <https://github.com/pingcap/docs/blob/master/etc/Syncer.json> still not updated?